components/Header.js

```diff
@@ -86,15 +97,41 @@ export default function Header() {
       </StyledHeader>
       {menuOpen && (
         <StyledNavigation>
+          {!session?.user ? (
+            // <StyledLink href="/login" $active={router.pathname === "/login"}>
+            //   <div onClick={handleMenuClose}>Login</div>
+            // </StyledLink>
+            <p></p>
```

Comment:
Iam not happy with empty HTML tags on main. maybe just give a hint to the user or ont render anything at all in the case the user is not logged in.